tachyon/__main__.py

```diff
@@ -145,31 +141,33 @@ def finish_output(print_worker):
         pass
 
 
-def main():
-    # Parse command line
-    from tachyon.core.arguments import generate_options, parse_args
```

Comment:
Is the file still in use? Shouldn't be removed as well?

Reply:
I will